makanan/utils.py

```diff
@@ -0,0 +1,41 @@
+import re
+
+
+# Method untuk mengembalikan data makanan
+def makanan_data(makanan, message=None):
+    data = {
+        "pk": makanan.pk,
+        "fields": {
+            "nama": makanan.nama,
+            "harga": makanan.harga,
+            "deskripsi": makanan.deskripsi,
+            "gambar": makanan.gambar.url if makanan.gambar else "",
+            "kalori": makanan.kalori,
+            "restoran": makanan.restoran.pk,
+            "kategori": makanan.kategori.pk,
```

Comment:
GAK BISA INI KAYAK GINI, KATEGORI ITU MANY TO MANY, JANGAN PLEK KETIPLEK, TES DULU LAH DI POSTMAN!!! INTERNAL SERVER ERROR INI!!!

Reply:
aight", lemme fix the demn code